clientv3/lease.go

```diff
@@ -57,8 +57,10 @@ type Lease interface {
 type lessor struct {
 	c *Client
 
-	mu     sync.Mutex       // guards all fields
-	conn   *grpc.ClientConn // conn in-use
+	mu      sync.Mutex       // guards all fields
+	conn    *grpc.ClientConn // conn in-use
+	initedc chan bool
```

Comment:
Instead of this, could `recvKeepAliveLoop` manage the lifetime of `sendKeepAliveLoop`? link goes down => send goroutine exits, recv loop sees conn failure since it's always listening, reconnects, then starts new sender goroutine (lifetime bound to the new stream) on success reconnect